app/src/main/java/com/kyuwon/booklog/service/session/AuthenticationService.java

```diff
@@ -31,4 +32,15 @@ public String login(UserLoginData userLoginData) {
 
         return webTokenUtil.encode(user.getId());
     }
+
+    /**
+     * 인증 토큰으로 id(회원 식별자)를 받아 리턴한다.
```

Comment:
이렇게 하는 것이 더 적절하지 않을까요?
```suggestion
     * 인증 토큰을 받아 id(회원 식별자)를 리턴한다.
```

Reply:
그렇네요! 수정했습니다!